app/models/mentor_profile.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MentorProfile < ApplicationRecord
+  has_paper_trail
+
+  belongs_to :user
+  belongs_to :school
+  belongs_to :core_induction_programme, optional: true
+  belongs_to :cohort, optional: true
+  has_many :early_career_teacher_profiles
+  has_many :early_career_teachers, through: :early_career_teacher_profiles, source: :user
+  # has_one :participation_record, dependent: :destroy
```

Comment:
Delete

Reply:
I left this in on purpose! It's to make it clear that it doesn't have it (but might need to in the future).

Reply:
Commented out code looks ugly :/ 
Can we add a comment with a TODO here? 
`# TODO: Add a link to participant_record if we need to`?

Reply:
@callumacrae Ah, I though that it is a testing leftover or something. In this case, yea, please add TODO

Reply:
done :)